lib/cog_api/client.ex

```diff
@@ -46,6 +46,8 @@ defmodule CogApi.Client do
   @callback relay_group_delete(String.t, %Endpoint{}) :: {atom, [String.t]}
   @callback relay_group_add_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
   @callback relay_group_remove_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
+  @callback relay_group_add_bundle(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
```

Comment:
Can you explain more about your issues?

I agree that we're probably not going to need it but also feel really weird about leaving in code that is untested and also something that is not mirrored in the `fake`.